frontend/src/main/web/index.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import Router from 'react-router';
+import RootContent from './lib/components/RootContent';
+import Views from './lib/constants/Views.js';
+import Configs from './lib/constants/Configs';
+import StringUtils from './lib/utils/StringUtils';
+import _ from 'lodash';
+import css from './index.css'
+
+var mountNode = document.getElementById('main-content'),
+  baseUrl = mountNode.getAttribute('base-url'),
+  user = JSON.parse(mountNode.getAttribute('user')),
+  data = JSON.parse(mountNode.getAttribute('data')),
+  view = Views.getView(data.view),
+  dev = data.dev;
```

Comment:
@aeng Looks like this is not linted with standardjs (since that wants a separate var statement for each variable). I think we will change everything to a config based on standardjs in the next few months. You could lint it against standardjs now if you want to reduce the amount of changes needed later in a reformat, but is it up to you. It would be quite a big set of changes whether done now or later, so maybe it is best to wait until the linting config is ready.